src/components/Hero.js

```diff
@@ -9,7 +9,7 @@ const Hero = () => {
         query {
             hero: file(relativePath: { eq: "hero.jpg" }) {
                 childImageSharp {
-                    fluid(maxWidth: 2000, maxHeight: 900) {
+                    fluid(maxWidth: 2000, maxHeight: 900, quality: 100) {
```

Comment:
bro why
theres no need, stop killing perf

Reply:
The hero image is NOTICABLY more pixelated without quality 100, so this might have to stay

Reply:
ig its fine for now